load-test/entrypoint.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash -l
+
+set -e
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+ITER8="/bin/iter8"
+
+echo "Creating working directory"
+
+WORK_DIR=`mktemp -d -p  "$DIR"`
+if [[ ! "$WORK_DIR" || ! -d  "$WORK_DIR" ]]; then
+  echo "Cound not create temporary working directory"
+  exit 1
+fi
+
+# no need to cleanup
+
+echo "Fetch load-test experiment"
+$ITER8 hub -e load-test
+cd load-test
+
+OPTIONS=""
```

Comment:
Is it possible to have a map[string]string as input in GH actions? And then simply iterate  over the keys and values of the map and set  them all? Instead of trying to statically check for values to be set?

Reply:
I read the input syntax. Looks like this is not possible.

Reply:
How about:

```yaml
- uses: iter8-tools/load-test@v1
  with:
    valuesFile: path/to/my-values.yaml
```

This would enable users to write (completely eliminating the need to escape special character .):
```yaml
- run: |
    cat << EOF > path/to/my-values.yaml
    url: https://example.com
    payloadURL: https://payload.com
    SLOs:
      p95.0: 200
      error-rate: 0
    EOF
- uses: iter8-tools/load-test@v1
  with:
    valuesFile: path/to/my-values.yaml
```

Reply:
> For CI/CD use-cases, it makes total sense for the user to check in the values file as part of the repo.
> 
> Of course, this would also enable users to write the following (creating file on the fly; also, completely eliminating the need to escape special character .):

I completely missed this on first reading. I agree. And the current implementation takes an input `values` that can be used in just this way. Question is ... **do we want to completely eliminate the other options**?

Doing so would simplify the implementation (no need to worry about these inputs) and also simplifies the documentation challenge. Currently, we require descriptions in `action.yaml` and documentation in `README.md` both of which must be consistent with the helm chart.

Reply:
I think `helm` style `--set` values is starting to make more and more sense for the command line, and less and less sense in this GitHub action, where `values.yaml` looks like the easier option all around (both for the end-user and for the maintainers).

Reply:
made these changes.